packages/server/graphql/queries/invoices.ts

```diff
@@ -54,8 +54,19 @@ export default {
         .orderBy(r.desc('startAt'), r.desc('createdAt'))
         .limit(first + 1)
         .run(),
-      after ? Promise.resolve(undefined) : makeUpcomingInvoice(orgId, stripeId)
+      r
+        .table('OrganizationUser')
+        .getAll(orgId, {index: 'orgId'})
+        .filter({
+          inactive: false,
+          removedAt: null
+        })
+        .count()
+        .run()
     ])
+    const upcomingInvoice = after
+      ? undefined
+      : await makeUpcomingInvoice(orgId, orgUserCount, stripeId)
```

Comment:
+1 can this be called in parallel with the calls above?

Reply:
No, it requires `orgUserCount` from those calls.